db/schema.rb

```diff
@@ -88,17 +87,19 @@
 
   create_table "oauth_applications", force: :cascade do |t|
     t.string   "name",                  limit: 255
-    t.string   "uid",                   limit: 255,                null: false
-    t.string   "secret",                limit: 255,                null: false
-    t.string   "redirect_uri",          limit: 255,                null: false
-    t.datetime "created_at",                                       null: false
-    t.datetime "updated_at",                                       null: false
+    t.string   "uid",                   limit: 255,                 null: false
+    t.string   "secret",                limit: 255,                 null: false
+    t.string   "redirect_uri",          limit: 255,                 null: false
+    t.datetime "created_at",                                        null: false
+    t.datetime "updated_at",                                        null: false
     t.string   "home_uri",              limit: 255
     t.string   "description",           limit: 255
     t.boolean  "supports_push_updates",             default: true
+    t.time     "deleted_at"
```

Comment:
We've still got this column, and the index on it, being added - we should get to the bottom of why these are still here and work out if they should be or shouldn't.  The right thing might be that we want these fields in the schema because they're in the production DB, but it'd be nice to investigate and have a separate commit that adds them with a message that explicitly addresses where they came from and why we're adding them without a migration.